### ISSUE TITLE
Failed to upload log, movescount.com replied with " "" " 

### ISSUE THREAD

**Issue by villeambit** [openambitproject/openambit#87]
when connecting the device via USB, openambit will not sync with movescount

In the terminal the following message occurs:

Failed to upload log, movescount.com replied with " "" " 

This is in openambit 0.3-1 for Debian, running on Ubuntu 14.04 LTS for AMD64

$ apt-cache policy openambit
openambit:
  Asennettu: 0.3-1
  Ehdokas:   0.3-1

$ uname -a
Linux ville 3.13.0-32-generic #57-Ubuntu SMP Tue Jul 15 03:51:08 UTC 2014 x86_64 x86_64 x86_64 GNU/Linux

$ lsb_release -rd
Description:    Ubuntu 14.04.1 LTS
Release:    14.04

**Comment by metbosch** [comment]
It looks like now moveslink is asking for username and password directly on the app. Instead of ask email amb later open the browser to allow the connection.

![untitled](https://cloud.githubusercontent.com/assets/4162025/5749059/7c98f99c-9c45-11e4-8203-6550e209ac20.png)

**Comment by VTiTux** [comment]
I have the same problem. Is there a solution?

**Comment by metbosch** [comment]
Unlinking openambit from movescount and re-configuring the program as a new installation worked for me.

**Comment by VTiTux** [comment]
1) Into movescount -> parameters -> connexions, I had 8 "moveslink2 "items. For all, I clicked to "ignored", but I can't completely remove them.
2) I removed the .openambit directory
3) I ran openambit and re-set my email into settings->movescount.
4) I clicked " Sync now", but the syncronisation failed, with this message into terminal: Failed to upload log, movescount.com replied with " "" " 

I haven't got any message of mouvescount about a new application connexion request.
What I miss?
